charts/zammad-ldap-sync/values.yaml

```diff
@@ -2,7 +2,7 @@ ldapSync:
   image:
     registry: ghcr.io
     repository: it-at-m/zammad-ldap-sync
-    pullPolicy: Always
+    pullPolicy: IfNotPresent
```

Comment:
For the ldap-sync we do not use version tags but environment tags (dev, integration, test, prod), which act like the 'latest' tag.

For `IfNotPresent` to be used effectively, the entire pipeline would first need to be restructured.